vuepress/docs/next/tutorials/micro-frontends/react.md

```diff
@@ -303,15 +301,17 @@ Note: You can drag and drop the files in your browser.
 <my-widget />
 ```
 
-4. Click `Save`.
+4. Click `Save`
+
+### View the Widget
 
-### View in Homepage
+Let's see the React micro frontend in action on your page.
 
-Let's see the React micro frontend in action on our page.
+1. In the `Entando App Builder` go back to `Pages` → `Management` 
```

Comment:
line 175 uses commas but other similar instructions do not. do you have a style preference we should follow?

Reply:
It's not really about preferences, for the most part. I think the recommendation is that longer introductory phrases with 6 or more words should have commas, unless there's complexity. There's always exceptions of course and I'm not an expert on commas by no means but I used it earlier because of the multiple steps. The addition of the comma broke those steps down more clearly I thought. In this case, there was only one step and less than 6 words.